lightblock.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/wire"
+)
+
+type LightBlock struct {
+	height  int32
```

Comment:
We can make this public

Reply:
Yes, but this is unnecessary. The reason here: https://github.com/gonative-cc/bitcoin-lightclient/pull/2#discussion_r1789711085.

Reply:
ok